setup.cfg

```diff
@@ -20,9 +20,11 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    pylint==1.7.6
-    pylint-django==0.7.2
-    pylint-celery==0.3
+    pylint==1.9.5;python_version<"3.0"  # The last pylint version to support python 2
+    pylint==2.4.2;python_version>="3.0"
+    pylint-django==0.11.1;python_version<"3.0"  # The last pylint-django to support python 2
+    pylint==2.0.11;python_version>="3.0"
```

Comment:
This line is a duplicate of line 24, but with a different version?

Reply:
oops. supposed to be `pylint-django`